### PR TITLE
Add api documentation to the doc tree

### PR DESCRIPTION
**Pull request by liyanhui1228**

### CODE REVIEW

File: trace/opencensus/trace/samplers/always_on.py [hunk 1]
```diff
@@ -16,7 +16,8 @@
 
 
 class AlwaysOnSampler(Sampler):
+    """Trace every requests."""
```
Comment (chingor13) [inline trace/opencensus/trace/samplers/always_on.py]: requests -> request

File: trace/opencensus/trace/samplers/probability.py [hunk 2]
```diff
@@ -35,6 +35,15 @@ def __init__(self, rate=None):
         self.rate = rate
 
     def should_sample(self, trace_id):
+        """Make the sampling decistion based on the lower 8 bytes of the trace
```
Comment (chingor13) [inline trace/opencensus/trace/samplers/probability.py]: desistion -> decision

File: trace/opencensus/trace/samplers/always_on.py (hunk 1 above)
Comment (liyanhui1228) [reply trace/opencensus/trace/samplers/always_on.py]: Fixed.

File: trace/opencensus/trace/samplers/probability.py (hunk 2 above)
Comment (liyanhui1228) [reply trace/opencensus/trace/samplers/probability.py]: Fixed.

**Comment by liyanhui1228** [comment]
@chingor13 Thanks for your review. I have tested locally and the generated doc page seems good. Will merge once the CI is green.

**Pull request merged**